init-tools.sh

```diff
@@ -22,47 +22,59 @@ case $OSName in
 
     Linux)
         OS=Linux
-        __DOTNET_PKG=dotnet-ubuntu-x64
+        source /etc/os-release
+        if [ $ID = "centos" ]; then
```

Comment:
According to http://www.tldp.org/LDP/abs/html/comparison-ops.html, `==` and `=` are equivalent for string comparison in Bash, although only `=` is supported in SH. I think it does make sense to use `==` since this is exclusively a Bash script.